src/app/views/sessions/signin/signin.component.html

```diff
@@ -99,6 +99,13 @@
             </div>
           </div></span>
 
+          <div class="truecommand">
```

Comment:
Maybe we should only show this if TC is configured/connected?

@kmoore134 @beanpole135 should we have a link to the TC wireguard IP here instead of the static portal link?

Reply:
The Wireguard endpoint is not useful to show at all. There is nothing there that the user can interact with. Showing the NAS's IP within the Wireguard network will probably be useful though.

Reply:
How so? Isnt the wireguard endpoint IP the IP of the truecommand instance? Its two ways, no?

Reply:
We should be showing the WG internal IP if we show anything, not the public endpoint IP. It should be showing 172.10.x.1 or similar which is the legit TC http/https webUI address.

Reply:
If we can't get the WG internal IP / network information, then no point in showing anything beyond portal link.